### PR TITLE
updating button styles

### PR DESCRIPTION
**Pull request by notandrew**

### CODE REVIEW

File: packages/react/src/themes/teams/components/Button/buttonStyles.ts [hunk 1]
```diff
@@ -43,25 +42,65 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, ButtonVa
...
       ...(!p.text && {
-        outline: 0,
-        borderWidth,
         borderStyle: 'solid',
         borderColor: v.borderColor,
         boxShadow: v.boxShadow,
+        borderRadius: '.2rem',
+        border: 0,
+
+        '::after': {
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Button/buttonStyles.ts]: what is the `after` and `before` here for?

Comment (notandrew) [reply packages/react/src/themes/teams/components/Button/buttonStyles.ts]: This is for applying the correct focus border styles to the buttons, one inner border and one outer border. When we use the normal border property, the content can shift, this just applied pseudo borders so the shape doesn't change.

File: packages/react/src/themes/teams/components/Button/buttonStyles.ts [hunk 2]
```diff
@@ -41,30 +40,73 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps, ButtonVariables> = {
...
+            content: '""',
+            borderRadius: '.2rem',
+            border: '.1rem solid transparent',
+            top: '.1rem',
+            right: '.1rem',
+            bottom: '.1rem',
+            left: '.1rem',
+          },
+
+          ...(p.isFromKeyboard && {
```
Comment (mnajdova) [inline packages/react/src/themes/teams/components/Button/buttonStyles.ts:77]: `isFromKeyboard` does not exists any more, use the":focus-visible" instead. Also, never use multiple pseudo selectors at the same time. You can change the following lines to:
```
":focus": {
  ":active": {
    borderColor: '...',
  },
}
```

File: packages/react/src/themes/teams/components/Button/buttonStyles.ts [hunk 3]
```diff
@@ -41,30 +40,73 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps, ButtonVariables> = {
...
+            ':focus::after': {
+              borderColor: '#252423',
```
Comment (mnajdova) [inline packages/react/src/themes/teams/components/Button/buttonStyles.ts:79]: No hex values please

File: packages/react/src/themes/teams/components/Button/buttonStyles.ts [hunk 4]
```diff
@@ -41,30 +40,73 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps, ButtonVariables> = {
...
+            },
+
+            ':focus::before': {
+              borderColor: '#fff',
```
Comment (mnajdova) [inline packages/react/src/themes/teams/components/Button/buttonStyles.ts:83]: No hex values please

**Pull request closed without merging**